packages/block-library/src/gallery/block.json

```diff
@@ -73,8 +29,25 @@
 		"sizeSlug": {
 			"type": "string",
 			"default": "large"
+		},
+		"allowAlign": {
+			"type": "boolean",
+			"default": false
+		},
+		"allowResize": {
+			"type": "boolean",
+			"default": false
+		},
+		"isList": {
+			"type": "boolean",
+			"default": true
 		}
 	},
+	"providesContext": {
+		"allowAlign": "allowAlign",
+		"allowResize": "allowResize",
+		"isList": "isList"
```

Comment:
Shouldn't this be named `isListItem`? Similar to how `allowAlign` is referring to the context receiver allowing/disallowing alignment (not whether the provider allows alignment), `isListItem` would refer to the receiver being a list item.

Reply:
Yeh, good point, i was looking at it from the wrong angle - have changed to isListItem as this makes more sense where the context is actually consumed